wifiphisher/extensions/karma.py

```diff
@@ -0,0 +1,108 @@
+"""
+Extension that check victims associated with rogue AP by KARMA attack
```

Comment:
I know that on `roguehostapd` we mention these data as `karma_data`. However, the truth is that the associated victims here may be the result of other attacks (e.g. the Known Beacons attack).

Instead of naming this extension `karma.py`, I think it's more suitable to name it `roguehostapdinfo.py`. It can work as a binding between Wifiphisher and `roguehostapd` in general.

I would also change the docstring to: " Extension that interacts with roguehostapd to print relevant information. For example, information regarding automatic association attacks."